storage-stackdriver/src/main/java/zipkin2/storage/stackdriver/StackdriverSpanConsumer.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.storage.stackdriver;
+
+import com.google.devtools.cloudtrace.v1.PatchTracesRequest;
+import com.google.devtools.cloudtrace.v1.Trace;
+import com.google.devtools.cloudtrace.v1.Traces;
+import com.google.protobuf.Empty;
+import io.grpc.CallOptions;
```

Comment:
you'll notice this doesn't pull in code like oauth libraries and netty. We might consider the former, but the latter makes it really version sensitive with the server (later this can be simply wrapped to create a zipkin reporter folks can put in apps.. might make sense to pre-ordain a zipkin-stackdriver-netty variant for those who can accept an externally provided netty+boringssl combo)